mayastor/src/subsys/nvmf/subsystem.rs

```diff
@@ -247,6 +248,9 @@ impl NvmfSubsystem {
 
     /// enable Asymmetric Namespace Access (ANA) reporting
     pub fn set_ana_reporting(&self, enable: bool) -> Result<(), Error> {
+        if std::env::var("NEXUS_NVMF_ANA_ENABLE").is_err() {
```

Comment:
Looks like this variable doesn't take into account the actual value of the variable, just the presence of the variable itself. Is that by design ?
I mean, whether its 'NEXUS_NVMF_ANA_ENABLE=1' or 'NEXUS_NVMF_ANA_ENABLE=0' - doesn't matter.
Shall we also enable/disable based on the actual value of the variable (0 - disable, 1 - enable) or so ?

Reply:
That was somewhat deliberate, it's only meant to be temporary (the Linux kernel NVMe target, for instance, has ANA reporting always enabled, AFAICT). I've changed it to require the env variable to be defined as "1".